packages/ckeditor5-table/src/plugin.ts

```diff
@@ -0,0 +1,139 @@
+/**
```

Comment:
As I commented in the PR thread, I'd consider renaming this to `package.ts` as it contains kind of metadata for a package. But then I'd merge the `fooconfig.ts` file into this one too. Why having configs in one place and plugin and command maps in another?

Reply:
On today's daily, the team agreed to use the `augmentation.ts` file name and to store config in a separate file.